src/db/tables/onerep_scans.js

```diff
@@ -0,0 +1,82 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ import initKnex from 'knex'
+ import knexConfig from '../knexfile.js'
+ const knex = initKnex(knexConfig)
+ 
+/**
+ * @param {number} onerepProfileId
+ * @param {number} oneRepScanId
+ * @returns {Promise<object>} oneRepScanResults
+ */
+async function getOnerepScanResults (onerepProfileId, oneRepScanId) {
+  return await knex('onerep_scans')
+    .select('onerep_scan_results')
+    .where('onerep_profile_id', onerepProfileId)
+    .andWhere('onerep_scan_id', oneRepScanId)
+}
+
+/**
+ * @param {number} onerepProfileId
+ * @returns {Promise<{created_at: number, updated_at: number, onerep_scan_results: object}[]>}
+ */
+async function getLatestOnerepScan (onerepProfileId) {
+  return await knex('onerep_scans')
+    .select('created_at', 'updated_at', 'onerep_scan_results')
+    .where('onerep_profile_id', onerepProfileId)
+    .orderBy('created_at', 'desc')
+    .limit(1)
+}
+
+/**
+ * @param {import('../../app/(nextjs_migration)/(authenticated)/user/breaches/breaches.js').Subscriber} subscriber
+ * @param {number} onerepProfileId
+ */
+async function setOnerepProfileId (subscriber, onerepProfileId) {
+  await knex('subscribers')
+    .where('id', subscriber.id)
+    .update({
+      onerep_profile_id: onerepProfileId,
+      // @ts-ignore knex.fn.now() results in it being set to a date,
+      // even if it's not typed as a JS date object:
+      updated_at: knex.fn.now(),
+    })
+}
+
+/**
+ * @param {number} onerepProfileId
+ * @param {number} oneRepScanId
+ */
+async function setOnerepScan (onerepProfileId, oneRepScanId) {
+  await knex('onerep_scans')
+    .insert({
+      onerep_profile_id: onerepProfileId,
+      onerep_scan_id: oneRepScanId,
+      created_at: knex.fn.now()
+    })
+}
+
+/**
+ * @param {number} onerepProfileId
+ * @param {number} oneRepScanId
+ * @param {object} oneRepScanResults
+ */
+async function setOnerepScanResults (onerepProfileId, oneRepScanId, oneRepScanResults) {
+  await knex('onerep_scans')
+    .where('onerep_profile_id', onerepProfileId)
+    .andWhere('onerep_scan_id', oneRepScanId)
+    .update({
+      onerep_scan_results: oneRepScanResults,
+      updated_at: knex.fn.now()
+    })
+}
+
+export {
+  getLatestOnerepScan,
```

Comment:
We should sync up on the verbiage for actions. 
So far we have inherited:
`get..`
`update...`
`add..`
`delete..`
`remove..`

and we are adding here:
`set...`

Reply:
Hm, `add` feels like it's be additive... do you have a specific proposal @mansaj, do you think this should be e.g. `addOnerepProfileId`?